analyses/sex-prediction-from-RNASeq/01-clean_split_data.R

```diff
@@ -101,6 +101,12 @@ option_list <- list(
     default = 0.7,
     help = "The proportion of samples that should be in the training set when splitting into training and testing sets",
     metavar = "double"
+  ),
+  optparse::make_option(
+    c("-e", "--target_columns"),
+    type = "character",
+    default = "reported_gender",
+    help = "Names of columns to use as targets in testing",
```

Comment:
```suggestion
    help = "Names of columns to use as targets in testing separated with a comma",
```
because it should be something like `reported_gender,germline_sex_estimate` correct?

Reply:
Yes.  The shell script  concatenates all columns specified and passes a single string with values separated by commas for this argument.

Will do in local repository and push.